app/frontend/src/components/Datepicker.tsx

```diff
@@ -66,6 +66,9 @@ export default function Datepicker({
             onChange={(date) => {
               if (date?.isValid()) onChange(date?.toDate());
             }}
+            openTo="year"
+            placeholder="dd/mm/yyyy"
```

Comment:
This isn't true for all locales.

I think a better solution instead of a placeholder just have `Datepicker` accept an optional `defaultValue` prop, and if undefined use today as a default value.

Reply:
I can't think of what a suitable `defaultValue` would be. I looked at something like 01/01/2000, but it feels clunkier compared to just using today's date. Whats the impact of omitting the default value all together?

Reply:
Not sure. Today is probably fine!